### PR TITLE
Fix typo: "/" -> "\"

### PR DESCRIPTION
**Pull request by exigow**

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Prerequisite: You have to be **signed in**. This is important because we expect
     System.out.
     ```
 2. Place a cursor at the end of the `System.out.` line.
-3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>/</kbd>.
+3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>\</kbd>.
```
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ Prerequisite: You have to be **signed in**. This is important because we expect
     public void bubbleSort(int[] array) {
     ```
 2. Place the cursor at the end of the line.
-3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>/</kbd>.
+3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>\</kbd>.
```
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ Prerequisite: You have to be **signed in**. This is important because we expect
     System.out.println("Hello World!");
     ```
 2. Place cursor at the end of the `// print ` line.
-3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>/</kbd>.
+3. Trigger autocompletion with <kbd>Alt</kbd> + <kbd>\</kbd>.
```
Comment (mkondratek) [inline TESTING.md]: pls mention macOS btw: `<kbd>option</kbd> + <kbd>\</kbd>`

File: TESTING.md (hunk 1 above)
Comment (exigow) [reply TESTING.md]: done

File: TESTING.md (hunk 3 above)
Comment (exigow) [reply TESTING.md]: done

**Comment by exigow** [comment]
Fixed all threads. Waiting for approval.

**Review by mkondratek — APPROVED** [review]

**Pull request merged**